routes/admin.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * To be mounted at /admin
+ */
+const express = require('express');
+const router = express.Router();
+
+const os = require('os');
+const ip = require('ip');
+
+const utils = require('../utils');
+const sdkModule = require('../sdk');
+const sdk = sdkModule.factory;
+
+/**
+ * ping method, just that.
+ */
+router.get('/ping', (req, res) => {
+  console.log('pong');
+  res.status(200).send('pong');
+});
+/**
+ * health check method. Will respond according to SDK status. Two options:
+ *    200 - Everything is OK. Ready to evaluate.
+ *    500 - Something is wrong or not ready yet. Not able to perform evaluations.
+ */
```

Comment:
503 is not possible also ?

Reply:
I was removing the function checking for that state but forgot a few lines. Thanks for the catch!

Reply:
As for the why it's not an option now, is because of the configuration option for BUR before spinning the server up. We should chat if this is really what we want on another time.

Reply:
My two cents here, this endpoint is used for 3rd party software to check if the service is available, so those applications try to get a 200 http status code, anything different to 200 means `service down`.
Conclusion, 200 it's ok and 500 (internal server error) it's ok as well from my point of view